### PR TITLE
Expand all issue in tasklist  fixed 

### PR DESCRIPTION
**Pull request by fahad-aot**
# Issue Tracking

JIRA: https://aottech.atlassian.net/browse/FWF-2831
Issue Type: BUG/ FEATURE

# Changes
1.Expand all for task variable issue fixed

# Screenshots (if applicable)

### CODE REVIEW

File: forms-flow-web/src/components/ServiceFlow/list/ServiceTaskListView.js [hunk 1]
```diff
@@ -113,9 +121,8 @@ const ServiceTaskListView = React.memo(() => {
         updatedExpandedTasks[task.id] = newExpandedState;
```
Comment (shuhaib-aot) [inline forms-flow-web/src/components/ServiceFlow/list/ServiceTaskListView.js:121]: what is newExpanded state

**Review by shuhaib-aot — CHANGES_REQUESTED** [review]
please fix he edge case also if there is no variable to show in expanded row then currently some blank space are showing right. so if there is no data is there stick on the current row no need to expand 
image

**Comment by fahad-aot** [comment]
> please fix he edge case also if there is no variable to show in expanded row then currently some blank space are showing right. so if there is no data is there stick on the current row no need to expand image

image
fixed

**Review by shuhaib-aot — APPROVED** [review]

**Review by abilpraju-aot — APPROVED** [review]

**Review by abilpraju-aot — COMMENTED** [review]
resolve conflicts

**Pull request merged**